src/test/java/org/swisspush/reststorage/mocks/FailFastVertxHttpServerResponse.java

```diff
@@ -29,8 +33,13 @@ public HttpServerResponse exceptionHandler(Handler<Throwable> handler) {
     }
 
     @Override
-    public HttpServerResponse write(Buffer data) {
-        throw new UnsupportedOperationException(msg);
+    public Future<Void> write(Buffer data) {
+        return null;
```

Comment:
Why don't we throw an Exception here?

Reply:
HINT: Counts for all methods in a class named "*fail-fast*" ;)

Reply:
ok